### PR TITLE
Feature/json ld

### PR DESCRIPTION
**Pull request by dana-ross**
For issue #168

### CODE REVIEW

**Comment by dana-ross** [comment]
I'd like another set of eyes on this before merging. Do any front-end engineers have time to review this?

File: _includes/markdown/Markup.md [hunk 1]
```diff
@@ -99,6 +99,7 @@ To this end, we use Schema.org markup where relevant and reasonable to ensure th
 
 Schema.org markup should be validated against the [Google Structured Data Testing Tool](https://developers.google.com/structured-data/testing-tool/).
 
+Schema.org markup should be validated against the [Google Structured Data Testing Tool](https://search.google.com/structured-data/testing-tool/u/0/).
```
Comment (jonathantneal) [inline _includes/markdown/Markup.md]: This looks so similar to the entry directly above it. Should both be present?

File: _includes/markdown/Markup.md [hunk 2]
```diff
@@ -89,17 +89,18 @@ Good:
...
 For example, if you've ever searched for a restaurant and found that it had star reviews in its search results, this is a product of Schema.org and rich snippets.
 
-Schema.org markup is intended to tell the search engines what your data *means*, not just what it says.
+Schema.org data is intended to tell the search engines what your data *means*, not just what it says.
 
-To this end, we use Schema.org markup where relevant and reasonable to ensure that our clients have the best search visibility that we can provide.
+To this end, we use Schema.org data where relevant and reasonable to ensure that our clients have the best search visibility that we can provide.
 
-Schema.org markup should be validated against the [Google Structured Data Testing Tool](https://developers.google.com/structured-data/testing-tool/).
+Schema.org data can be provided in two forms: "microdata" markup added to a page's structure or a JSON-LD format. Google prefers developers adopt JSON-LD, which isolates the data provided for search engines from the markup meant for user agents.
```
Comment (jonathantneal) [inline _includes/markdown/Markup.md]: "*"microdata" markup added to a page's structure or a JSON-LD format*" reads odd.

Maybe "*microdata, which is added to a page's markup, or as JSON-LD*"?

**Comment by jonathantneal** [comment]
Should there be guidance on where to include the ld+json? Is it strictly included with an inline `<script>` tag, or can/should it be linked to as an external file?

File: _includes/markdown/Markup.md (hunk 1 above)
Comment (dana-ross) [reply _includes/markdown/Markup.md]: I'm gonna blame VS Code's chunk staging UI. Looks like the new paragraph got staged but the removal was wrapped up in the second commit.

File: _includes/markdown/Markup.md (hunk 2 above)
Comment (dana-ross) [reply _includes/markdown/Markup.md]: 👍

**Comment by dana-ross** [comment]
I don't believe any of the big search engines support JSON-LD in a `<link>` tag yet. All of Google's docs specify a `<script>` tag, and the HTML specs say the data has to be inline if you're using a `<script>` tag for data.

**Pull request merged**